purview/models/errata.py

```diff
@@ -32,6 +31,7 @@ class Advisory(PurviewStructuredNode):
     assigned_to = RelationshipTo('.user.User', 'ASSIGNED_TO', cardinality=ZeroOrOne)
     attached_bugs = RelationshipTo('.bugzilla.BugzillaBug', 'ATTACHED')
     attached_builds = RelationshipTo('.koji.KojiBuild', 'ATTACHED')
+    attached_products = RelationshipTo('.product.Product', 'ATTACHED')
```

Comment:
I think this should just be called `product` and it should have cardinality of `ZeroOrOne` if I'm not mistaken.

Reply:
I think it can be plural because one advisory can be attached to multiple products right.